test/JDBC/expected/BABEL-4046.out

```diff
@@ -0,0 +1,197 @@
+create table options(name varchar(30) collate bbf_unicode_cp1_ci_as)
```

Comment:
Could you use T-SQL collation instead of BBF's? I'd like to make test files to be executed in SQL Servers as much as possible.

Reply:
I can delete collate bbf_unicode_cp1_ci_as in this case, since it's the default collation